src/mongo/util/text.cpp

```diff
@@ -154,10 +155,8 @@ long long parseLL(const char* n) {
     long long ret;
     uassert(13307, "cannot convert empty string to long long", *n != 0);
 #if !defined(_WIN32)
-    char* endPtr = 0;
-    errno = 0;
-    ret = strtoll(n, &endPtr, 10);
-    uassert(13305, "could not convert string to long long", *endPtr == 0 && errno == 0);
+    const Status parse_res = parseNumberFromString<long long>(n, &ret);
+    uassert(13305, "could not convert string to long long", parse_res == Status::OK());
```

Comment:
uassert(13305, parse_res.toString(), parse_res.isOK());

Then, we get the error message out of the bad status in the exception that is thrown.